### PR TITLE
[12.0][MIG] stock_inventory_preparation_filter

### PR DESCRIPTION
**Pull request by xavierpiernas**
Standard migration

### CODE REVIEW

**Comment by rousseldenis** [comment]
@xavierpiernas Travis is red

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

File: stock_inventory_preparation_filter/models/stock_inventory.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+# Copyright 2015 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class StockInventoryEmptyLines(models.Model):
+    _name = 'stock.inventory.line.empty'
+
```
Comment (rousseldenis) [inline stock_inventory_preparation_filter/models/stock_inventory.py:12]: Please add a _description

**Review by HviorForgeFlow — APPROVED** [review]
Functional review LGTM :+1:

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**